app/app.js

```diff
@@ -152,7 +152,7 @@ module.exports = (options) => {
     const componentName = req.params.component
     const requestedExampleName = req.params.example || 'default'
 
-    const previewLayout = res.locals.componentData.previewLayout || 'layout'
+    let previewLayout = res.locals.componentData.previewLayout
```

Comment:
Non-blocking but as this variable is now used in the [iframe if-block](https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/components/fieldset/fieldset.yaml#L39) we could consider moving it above that block.

Reply:
@hannalaakso Yep sounds good too, all changed